lib/licensee/projects/git_project.rb

```diff
@@ -5,6 +5,9 @@
 # Project files for this project type will contain the following keys:
 #  :name - the file's path relative to the repo root
 #  :oid  - the file's OID
+
+autoload :Rugged, 'rugged'
+
 module Licensee
   module Projects
     class GitProject < Licensee::Projects::Project
```

Comment:
@@ -1,7 +1,6 @@
require_relative 'licensee/version'
require 'forwardable'
require 'pathname'
require 'rugged'
require 'yaml'

module Licensee
 3  lib/licensee/projects/git_project.rb 
Viewed
@@ -5,6 +5,9 @@
# Project files for this project type will contain the following keys:
#  :name - the file's path relative to the repo root
#  :oid  - the file's OID

autoload :Rugged, 'rugged'

module Licensee
  module Projects
    class GitProject < Licensee::Projects::Project